docs/properties.md

```diff
@@ -0,0 +1,127 @@
+# Properties
+
+> Synonyms: design tokens, design variables, design constants, atoms
+
+Style properties are stored in a collection of JSON or JS module files. We usually keep them in a `properties` directory, but you can put them wherever you like, they just need to be referenced in the `source` attribute on your `config.json` file.
+
+A property is a collection of attributes that describe any fundamental/atomic visual style. Each attribute is a `key:value` pair. A property name and its value are considered a design token (or design variable/constant/atom).
+
+![Terminology for different parts of a JSON property](assets/property-definitions.png)
+
+A property is transformed for use in different platforms, languages, and contexts. A simple example is a color. A color can be represented in many ways, all of these are the same color: `#ffffff`, `rgb(255,255,255)`, `hsl(0,0,1)`.
+
+A property file organizes properties in a structured way for easy access. Property files are organized as a deep object with the leaf nodes being the style key:value pairs.
+
+## Examples
+
+```json
+{
+  "color": {
+    "font": {
+      "base": { "value": "#111111" },
+      "secondary": { "value": "#333333" },
+      "tertiary": { "value": "#666666" },
+      "inverse": {
+        "base": { "value": "#ffffff" }
+      }
+    }
+  }
+}
+```
+
+Any object in the JSON that has a `value` attribute on it is a property, so in this example there are 4 style properties: `color.font.base`, `color.font.secondary`, `color.font.tertiary`, and `color.font.inverse.base`.
+
+For any properties you wish to output, the "value" attribute is required. This provides the data that will be used throughout the build process (and ultimately used for styling in your deliverables). You can optionally include any custom attributes you would like (e.g. "comment" with a string or "metadata" as an object with its own attributes).
+
+# Example Property
+Here you can see a property of "size.font.small" with two attributes:
+1. the required "value" attribute, set to "10"
+1. the optional "comment" attribute (The "comment" attribute is treated in a special way - the comment will appear in output files when the output format supports comments.)
+```json
+{
+  "size": {
+    "font": {
+      "small" : {
+        "value": "10",
+        "comment": "the smallest font allowed for readability"
+      },
+    }
+  }
+}
+```
+
+# Multiple Properties "size.font.\*"
+Multiple properties in a single file are simple to read and understand using the [`Category / Type / Item (CTI)`](#category-type-item-(cti)) method
```

Comment:
Maybe add the caveat here that this structure is not required, you can structure your properties however you want?

Reply:
Called it 'recommended'